lib/node-utils/index.ts

```diff
@@ -95,6 +96,20 @@ export function findClosestVariableDeclaratorNode(
 	return findClosestVariableDeclaratorNode(node.parent);
 }
 
+export function findClosetFunctionExpression(
+	node: TSESTree.Node | undefined
+): TSESTree.ArrowFunctionExpression | TSESTree.FunctionExpression | null {
+	if (!node) {
+		return null;
+	}
+
+	if (isArrowFunctionExpression(node) || isFunctionExpression(node)) {
```

Comment:
Should this also check FunctionDeclarations? It's a little silly, but someone could do something like:

```js
function a () {
  userEvent.click(blah);
}

test("blah", a);
```